CHANGELOG.md

```diff
@@ -33,6 +33,9 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 
 * Added a way to rename columns in the table macro with `#[sql_name="the_column_name"]`
 
+* `diesel print-schema` and `infer_schema!` are now both auto renaming columns
+  that are conflicting by adding a `_` at the end of the name
```

Comment:
How about

> `infer_schema!` now automatically renames columns that conflict with a Rust keyword by placing a `_` at the end of the name. For example, a column called `type` will be referenced as `type_` in Rust.

I don't think we need to mention `diesel print-schema` separately since it's documented as behaving the same as `infer_schema!`